backend/src/domain/models/Game.js

```diff
@@ -55,20 +55,13 @@ export class Game {
    * of the session
    */
   async nextRound() {
-    this.round++;
```

Comment:
Redundant code as this function is only called once. Thus round will always be at 1.